src/test/utils/meta_table_manager_test.cpp

```diff
@@ -0,0 +1,53 @@
+#include "../base_test.hpp"
+
+#include "storage/chunk_encoder.hpp"
+#include "utils/load_table.hpp"
+#include "utils/meta_table_manager.hpp"
+
+namespace opossum {
+
+class MetaTableManagerTest : public BaseTest {};
+
+TEST_F(MetaTableManagerTest, TableBasedMetaData) {
+  // This tests a bunch of meta tables that are somehow related to the tables stored in the StorageManager.
+  auto& storage_manager = StorageManager::get();
+
+  const auto prefix = std::string{MetaTableManager::META_PREFIX};
+  const auto path = std::string{"resources/test_data/tbl/meta_tables/meta_"};
+  for (const auto& meta_table_name : {"tables", "columns", "chunks", "segments"}) {
+    SCOPED_TRACE(meta_table_name);
```

Comment:
I haven't seen this before, what is it?

Reply:
If you have a test with expectations that are part of a loop, it gives you more context:

https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#adding-traces-to-assertions

Reply:
Ah, I see.